prow/plugins/override/override_test.go

```diff
@@ -597,6 +597,27 @@ func TestHandle(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:      "override with allow_top_level_owners works for uppercase user",
```

Comment:
Whats with the other way round, the user in the owners file is not all lowercase?

Reply:
The test function currently differs from the tested code in that aspect. The real code effectively runs `github.NormLogin` on all users which are loaded from the _OWNERS_ file. The test doesn't do that. Being aware of that we don't need to test the case you're mentioning, although I'm open to rewriting the test function to be more similar to the tested code.

Reply:
okay sgtm. Can youy sign the CLA and squash your commits?

Reply:
I squashed the first two commits and added a fixup commit on top that makes the changes to the test function as described above. I think it makes sense to keep that commit and squash it as well, WDTY?
Meanwhile I'll read up on the CLA signature.

Reply:
What is a fixup commit? I don't think there is value in having the testcase declare an uppercase user just so the test definition normalizes it, that is effectively just testcode testing testcode

Reply:
> What is a fixup commit?

It's what I call a commit that was created using `git commit --fixup=<some-rev>`.

> I don't think there is value in having the testcase declare an uppercase user just so the test definition normalizes it, that is effectively just testcode testing testcode

I see that point. Unfortunately the real code is not suited to be called from the test function and I'm currently not motivated to refactor that. I'll drop the fixup commit then.